package.json

```diff
@@ -19,6 +19,8 @@
     "npm": ">=7"
   },
   "dependencies": {
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.6.0",
```

Comment:
🤔 Should these also be `devDependencies`?

Reply:
That's my bad, I pushed more focus to migration of `eslint` configuration. It's fixed now.